src/main/java/com/artipie/rpm/pkg/FilePackage.java

```diff
@@ -183,4 +158,53 @@ public String href() {
             };
         }
     }
+
+    /**
+     * {@link AbstractHeader.Entry} based MetaHeader.
+     *
+     * @since 1.0
+     */
+    private static final class EntryHeader implements MetaHeader {
+
+        /**
+         * Native header entry.
+         */
+        private final Optional<AbstractHeader.Entry<?>> entry;
+
+        /**
+         * Ctor.
+         * @param entry Native header entry
+         */
+        EntryHeader(final Optional<AbstractHeader.Entry<?>> entry) {
```

Comment:
@victornoel let's add a ctor that takes `AbstractHeader` and wraps it to optional to make it more user friendly. Than we can remove ugly `Optional.ofNullable` from `FilePackage.header()` method.